Firestore/CHANGELOG.md

```diff
@@ -6,6 +6,7 @@
 - [feature] Change Firestore's Swift Package Manager distribution from source
   to binary to reduce the time it takes to add the Firebase package and to
   build the Firestore SDK (#6564).
+- [fixed] Fixed SSL symbol collision issue affecting SwiftPM users. (#6869)
```

Comment:
@morganchen12. this 10.**8**.0 entry was missed during that release and it being done so now. Could you regenerate the entry for that release?